pgcli/packages/parseutils.py

```diff
@@ -66,7 +66,9 @@ def last_word(text, include='alphanum_underscore'):
 
 TableReference = namedtuple('TableReference', ['schema', 'name', 'alias',
                                                'is_function'])
-TableReference.ref = property(lambda self: self.alias or self.name)
+TableReference.ref = property(lambda self: self.alias or (
```

Comment:
Not necessary for this PR, but long-term if this gets any more complicated we should think about replacing the namedtuple based class with an actual standalone classdef with docstrings etc.